### PR TITLE
Remove markmail search

### PR DESCRIPTION
**Pull request by AstraLuma**
Markmail is apparently dead, so remove the search box

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -475,19 +475,6 @@ <h2 class="icon icon-mailing-lists">Mailing Lists</h2>
 
       <div class="corner"></div>
 
-      <br/>
-
-      <div class="list-header">
-        <a href="https://mail-archives.apache.org/mod_mbox/couchdb-announce/"><strong>Mail Search</strong></a>
-        <p>Search all of our mailing lists at once.</p>
-      </div>
```
Comment (nickva) [inline index.html:483]: https://lists.apache.org/list.html?announce@couchdb.apache.org is still there it seems?

Comment (AstraLuma) [reply index.html:483]: 1. It's a much more confusing entry without the form following it.
2. The link is a lie (it only searches the announce list)

Comment (nickva) [reply index.html:483]: Ah, you're right. Good point.

**Review by nickva — APPROVED** [review]
+1

Thanks, @AstraLuma

**Pull request merged**